### ISSUE TITLE
scan-object-acl fails with traceback

### ISSUE THREAD

**Issue by rdonkin** [ankane/s3tk#15]
This looks like a really useful tool, thanks for writing and maintaining it!

I built a Docker image from a Git checkout and got this on `scan-object-acl`, for a bucket that exists:

```
$ docker run  -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY s3tk s3tk scan-object-acl foobucket

joblib.externals.loky.process_executor._RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/joblib-0.12.5-py3.7.egg/joblib/externals/loky/process_executor.py", line 420, in _process_worker
    r = call_item.fn(*call_item.args, **call_item.kwargs)
  File "/usr/local/lib/python3.7/site-packages/joblib-0.12.5-py3.7.egg/joblib/_parallel_backends.py", line 563, in __call__
    return self.func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/joblib-0.12.5-py3.7.egg/joblib/parallel.py", line 261, in __call__
    for func, args, kwargs in self.items]
  File "/usr/local/lib/python3.7/site-packages/joblib-0.12.5-py3.7.egg/joblib/parallel.py", line 261, in <listcomp>
    for func, args, kwargs in self.items]
  File "/usr/local/lib/python3.7/site-packages/s3tk-0.2.0-py3.7.egg/s3tk/__init__.py", line 172, in scan_object
    obj = s3.Object(bucket_name, key)
NameError: name 's3' is not defined
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/bin/s3tk", line 4, in <module>
    __import__('pkg_resources').run_script('s3tk==0.2.0', 's3tk')
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 661, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1441, in run_script
    exec(code, namespace, namespace)
  File "/usr/local/lib/python3.7/site-packages/s3tk-0.2.0-py3.7.egg/EGG-INFO/scripts/s3tk", line 7, in <module>
    s3tk.cli()
  File "/usr/local/lib/python3.7/site-packages/Click-7.0-py3.7.egg/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/Click-7.0-py3.7.egg/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/Click-7.0-py3.7.egg/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/Click-7.0-py3.7.egg/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/Click-7.0-py3.7.egg/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/s3tk-0.2.0-py3.7.egg/s3tk/__init__.py", line 489, in scan_object_acl
    summarize(parallelize(bucket, only, _except, scan_object))
  File "/usr/local/lib/python3.7/site-packages/s3tk-0.2.0-py3.7.egg/s3tk/__init__.py", line 267, in parallelize
    return Parallel(n_jobs=24)(delayed(fn)(bucket.name, os.key, *args) for os in objects if object_matches(os.key, only, _except))
  File "/usr/local/lib/python3.7/site-packages/joblib-0.12.5-py3.7.egg/joblib/parallel.py", line 996, in __call__
    self.retrieve()
  File "/usr/local/lib/python3.7/site-packages/joblib-0.12.5-py3.7.egg/joblib/parallel.py", line 899, in retrieve
    self._output.extend(job.get(timeout=self.timeout))
  File "/usr/local/lib/python3.7/site-packages/joblib-0.12.5-py3.7.egg/joblib/_parallel_backends.py", line 517, in wrap_future_result
    return future.result(timeout=timeout)
  File "/usr/local/lib/python3.7/concurrent/futures/_base.py", line 432, in result
    return self.__get_result()
  File "/usr/local/lib/python3.7/concurrent/futures/_base.py", line 384, in __get_result
    raise self._exception
NameError: name 's3' is not defined
```

Other commands do work OK:

```
$ docker run  -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY s3tk s3tk scan foobucket
foobucket
  ✔ ACL not open to public
  ✔ Policy not open to public
  ✘ Logging disabled
  ✔ Versioning enabled
  ✘ Default encryption disabled
```
The credentials I am using have admin access.

I had the same error with a virtualenv using brew Python 3.7.0 on Mac.

**Comment by ankane** [comment]
Hey @rdonkin, thanks for reporting 👍 It looks like s3tk doesn't work with the latest version of joblib. I've locked down the dependency on master for now.

**Comment by ankane** [comment]
Just published 0.2.1

**Comment by rdonkin** [comment]
Thanks for fix - however, commit 4253e03c has reverted the version spec on `joblib` in `setup.py`.

Not sure if that was intentional, will try a version without that.
